code/MemberImportController.php

```diff
@@ -0,0 +1,59 @@
+<?php
+
+namespace SilverStripe\Admin;
+
+use SilverStripe\Control\Controller;
+use SilverStripe\Security\Permission;
+use SilverStripe\View\Requirements;
+
+class MemberImportController extends Controller
```

Comment:
Moved logic off SecurityAdmin to a new controller because ModelAdmin isn't allowed to have random controller endpoints on it, as every subpath on a ModelAdmin is assumed to be a managed_model